modules/installation-osp-enabling-swift.adoc

```diff
@@ -17,6 +17,11 @@ If link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform
 If Swift is present and you want to use it, you must enable access to it. If it is not present, or if you do not want to use it, skip this section.
 ====
 
+[IMPORTANT]
+====
+Early versions of {rh-openstack} 17.0 set `rgw_max_attr_size` parameter of Ceph RGW to 256. This setting causes issues with uploading container images to the {product-title} registry. `rgw_max_attr_size` should be set to 1024. Before installation please check if your {rh-openstack} is affected by this problem and if so reconfigure Ceph RGW.
```

Comment:
```suggestion
Early versions of {rh-openstack} 17.0 set the `rgw_max_attr_size` parameter of Ceph RGW to 256 ${UNITS}. This setting causes issues with uploading container images to the {product-title} registry. You must set the value of `rgw_max_attr_size` to [at least?] 1024. 

Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.
```

Can we be more specific than "early?"

Reply:
"Early" in this case means that [1] is not yet fixed, but there's a consensus that it should be. I'm not sure how to handle this, maybe we just drop "early" and assume every deployment is affected?

Should I replace `${UNITS}` with something here? In this case it's string length, so "characters"?

[1] https://bugzilla.redhat.com/show_bug.cgi?id=2167161

Reply:
1. Fine by me. 

2. Yeah, and "characters" is fine. We just need to avoid ambiguity as much as possible for translation's sake.

Reply:
Alright, should be fixed now.